### DISCUSSION TITLE
Change storage driver from overlay2 to devicemapper

### DISCUSSION THREAD

**Discussion by thirumalaicb** [rancher-sandbox/rancher-desktop#1910] [Q&A]
Would it be possible to change the storage driver from overlay2 to **devicemapper**?  I need this as my containers require more storage space.

I referred to the [docker documentation](https://docs.docker.com/storage/storagedriver/device-mapper-driver/) and created a JSON file in **/etc/docker/daemon.json** (I also had to create the **docker** folder inside /etc) as below.
`{`
`  "storage-driver": "devicemapper",`
`  "storage-opts": [`
`    "dm.basesize=20G"`
`  ]`
`}`

But the docker info still reflects the storage driver to be overlay2.  FYI, I have also tried factory reset, uninstall/reinstall, before removing all the containers and images.

**Comment by Stargator** [comment]
I am similarly interested in this. @mook-as does RD plan to support this?
